multimap/src/main/java/org/infinispan/multimap/impl/SortedSetBucket.java

```diff
@@ -120,6 +121,39 @@ public Collection<ScoredValue<V>> inter(Collection<ScoredValue<V>> inputValues,
       return sortedMergeScoredValues;
    }
 
+   public List<ScoredValue<V>> randomMembers(int count) {
+      if (count == 1 || count == -1) {
```

Comment:
We could add something similar to this check but for `count >= entries.size() || Math.abs(count) >= entries.size()`.

Reply:
which benefit would we have ?

Reply:
Oh, I meant that we should return the complete list when the count is greater than the total size.